### ISSUE TITLE
Backward incompatibility in edge protocol

### ISSUE THREAD

**Issue by redlicha** [openvstorage/volumedriver#214]
Observed while testing a dev build, very likely introduced with #203 (part of #146) . Either backward compatibility needs to be restored or the breakage needs to be documented / a new major version for the client lib introduced.
@wimpers , @saelbrec : this needs to happen before releasing RC2.

**Comment by pploegaert** [comment]
I'd prefer to have this fixed ...

**Comment by wimpers** [comment]
Updating the edge client and server at the same time is required.

**Comment by redlicha** [comment]
@wimpers: IOW accept the breakage and bump the library version?

**Comment by cnanakos** [comment]
What is the problem bumping the client lib version and install the latest and greatest edge client package?

**Comment by wimpers** [comment]
I propose to bump the client lib version and install the latest and greatest edge client package. Hence this doesn't require any work from the voldrv team and can be closed with won't fix. 

@saelbrec if you disagree please re-open and assign to the right person.

**Comment by redlicha** [comment]
Reopening and assigning to @cnanakos as the library version still has to be bumped.

**Comment by dejonghb** [comment]
You guys are aware that clients (for example kernsafe iscsi setups) can be on completly different systems that are totally unware of anything changing at the server (openvstorage) side, right?

**Comment by pploegaert** [comment]
does this also require restarting certain components?  As this might not always that easy on a production environment.

Perhaps we can have the client lib log/return that it is incompatible with the server version, to avoid losing time debugging these incompatibilities

**Comment by wimpers** [comment]
Closed by PM.
